stingray/crossspectrum.py

```diff
@@ -638,11 +647,11 @@ class AveragedCrossspectrum(Crossspectrum):
 
     Parameters
     ----------
-    lc1: :class:`stingray.Lightcurve` object OR iterable of :class:`stingray.Lightcurve` objects
+    lc1: :class:`stingray.Lightcurve`object OR iterable of :class:`stingray.Lightcurve` objects OR :class:`stingray.EventList` object
         A light curve from which to compute the cross spectrum. In some cases, this would
         be the light curve of the wavelength/energy/frequency band of interest.
 
-    lc2: :class:`stingray.Lightcurve` object OR iterable of :class:`stingray.Lightcurve` objects
+    lc2: :class:`stingray.Lightcurve`object OR iterable of :class:`stingray.Lightcurve` objects OR :class:`stingray.EventList` object
```

Comment:
So I must admit I'm finding it quite confusing now that `lc1` and `lc2` can be either a `Lightcurve` object or an `EventList` object, especially since somewhere in the code we go from "lc1 is an EventList" to "lc1 is now a LightCurve".

 I'm wondering whether EventList-to-PSD may be better handled in an external function, rather than overloading `lc1` and `lc2`? This seems like it might be a fairly straightforward function two write. Otherwise we might want to consider renaming these two variables, which would be a major undertaking and break a lot of things.